scripts/tasks/webpack.js

```diff
@@ -19,3 +19,25 @@ exports.webpackDevServer = async function() {
     execSync(`node ${webpackDevServerPath} --config ${configPath} --port ${port} --open`);
   }
 };
+
+exports.webpackDevServerWithCompileResolution = async function() {
```

Comment:
Would you mind explaining more about what this does? (ideally in a code comment)

Reply:
```suggestion
/**
  * Launches webpack-dev-server that allows subsequent tasks to start only after the first successful build
  */
exports.webpackDevServerWithCompileResolution = async function() {
```